lib/src/lightning-manager.ts

```diff
@@ -694,6 +802,9 @@ class LightningManager {
 					peers: await this.getPeers(),
 					watch_outputs: [], //TODO add here when outputs are persisted
```

Comment:
Can we remove `watch_outputs` and `watch_transactions` from the backups here? Because we have `confirmed_outputs` and `confirmed_transactions` right?

Reply:
We should be good to remove `watch_outputs` & `watch_transactions` from backups.